lib/android.js

```diff
@@ -34,30 +35,33 @@ class JavaOnPathCheck extends DoctorCheck {
 class AndroidToolCheck extends DoctorCheck {
   constructor () {
     super();
-    this.tools = ['adb', 'android', 'emulator'];
+    this.tools = ['adb', 'android', 'emulator', 'apkanalyzer'];
```

Comment:
which apkanalyzer do we check here? appium-adb directly uses the jar package instead of the wrapper script

Reply:
I picked it from https://github.com/appium/appium-adb/blob/06ecb6e960de0c5fff566268dba6e63d2b25176b/lib/tools/system-calls.js#L65 I thought it was missed

Reply:
it is there, but is not used anywhere anymore.

Reply:
https://github.com/appium/appium-adb/blob/df80b6a4a61810131bb95ccb20c3f2ab035c802e/lib/helpers.js#L190

Reply:
ah, gotcha